.travis.yml

```diff
@@ -26,10 +26,10 @@ script:
   - yarn
   - ../../bin/graph codegen --debug
   - ../../bin/graph build --debug
-  # Exercise `graph test` with another example subgraph
-  - cd ../../examples/basic-event-handlers
+  # Exercise `graph local` with another example subgraph
+  - cd ../basic-event-handlers
```

Comment:
Why this changed?

Reply:
Well I thought since both `example-subgraph` and `basic-event-handlers` folders are in the `examples` folder there is no need to go out of the `examples` folder just to reenter it. I just did it as a micro readability improvement. Is there a drawback to doing it?

Reply:
Oh I see, okay, no problem!